test/TestPolarizationCalculator.cc

```diff
@@ -0,0 +1,825 @@
+/* This file is part of the CARTA Image Viewer: https://github.com/CARTAvis/carta-backend
```

Comment:
As in the case of #1016, I would like to split out this test code into a separate PR.